worlds/tunic/docs/en_TUNIC.md

```diff
@@ -86,3 +86,5 @@ Notes:
 - There is no limit to the number of Shops hard-coded into place.
 - If you have more than one shop in a scene, you may be wrong warped when exiting a shop.
 - If you have a shop in every scene, and you have an odd number of shops, it will error out.
+
+See the [Archipelago Plando Guide](https://archipelago.gg/tutorial/Archipelago/plando/en) for more information on Plando and Connection Plando.
```

Comment:
I'd probably recommend changing the URL to be a relative one instead of an absolute one otherwise, it won't route to the beta site version if hosted on beta site for example, or another webhost at another domain.